src/support/slack/commands/onboard.js

```diff
@@ -84,21 +87,57 @@ function OnboardCommand(context) {
     imsOrgID,
     configuration,
     profileName,
+    workflowWaitTime,
     slackContext,
   ) => {
     const { say } = slackContext;
+    const sfnClient = new SFNClient();
 
     const baseURL = extractURLFromSlackInput(baseURLInput);
 
+    log.info(`Slack context: ${JSON.stringify(slackContext)}`);
+
+    // Check if site is in LA_CUSTOMERS list
+    if (env.LA_CUSTOMERS) {
```

Comment:
Pls check with @ravkiran and @solaris007 on the best way to store the information about the LA customers.